project_one/script.js

```diff
@@ -0,0 +1,18 @@
+var button1 = document.getElementById("grading-element-one");
+var link = document.getElementById("grading-element-two");
+var button2 = document.getElementById("grading-element-three");
+var span = document.getElementById("grading-element-three-consequence");
+var ul = document.getElementById("grading-element-one-consequence")
+
+
+
+button1.addEventListener("click", function() {
+	var li = document.createElement("li");
```

Comment:
close but not quite...can you create the `ul` as part of the function passed to `button1.addEventListener("click", ` ? i should run `document.querySelector(#grading-element-one-consequence)` and get `null` until after I run `document.querySelector("#grading-element-one").click()`